packages/block-library/src/navigation/index.php

```diff
@@ -157,13 +157,19 @@ function render_block_core_navigation( $content, $block ) {
 		return '';
 	}
 
+	$orientation_class_name = '';
+	if ( isset( $attributes['orientation'] ) && 'vertical' === $attributes['orientation'] ) {
+		$orientation_class_name = 'is-vertical';
+	}
+
 	$colors          = block_core_navigation_build_css_colors( $attributes );
 	$font_sizes      = block_core_navigation_build_css_font_sizes( $attributes );
 	$classes         = array_merge(
 		$colors['css_classes'],
 		$font_sizes['css_classes'],
 		array( 'wp-block-navigation' ),
 		isset( $attributes['className'] ) ? array( $attributes['className'] ) : array(),
+		array( $orientation_class_name ),
```

Comment:
It seems when orientation class name is not vertical we are not passing an empty array but an array with an empty string. This adds an unexpected space on the class attribute "wp-block-navigation ". What if we simplify things a little and replace `array( $orientation_class_name )`with  `isset( $attributes['orientation'] ) && 'vertical' === $attributes['orientation'] ) ? array( 'is-vertical' ) : array(),`?

Reply:
Thanks @jorgefilipecosta I'll update accordingly.